lib/material_progress/_mixins.scss

```diff
@@ -29,3 +29,22 @@
     }
   }
 }
+
+// Changes the height and roundedness of the ends of <material-progress>.
+//
+// Example use:
+// :host material-progress {
+//   wide-rounded-progress-bar($mat-grid * 2);
+// }
+//
```

Comment:
Not sure we actually process "doc comments", but you're using `///` elsewhere, maybe you want that here too?

Reply:
We are planning on adding some doc comment like generation tool. There will need to be a cleanup fixit when we have something to build and look at.